Crane/buildSrc/src/main/java/com/example/crane/buildsrc/Dependencies.kt

```diff
@@ -21,7 +21,7 @@ object Versions {
 }
 
 object Libs {
-    const val androidGradlePlugin = "com.android.tools.build:gradle:7.0.0-beta05"
+    const val androidGradlePlugin = "com.android.tools.build:gradle:7.0.0-rc01"
```

Comment:
Please can we revert this AGP upgrade unless there's a specific need? We prefer to keep all the samples in sync and upgrade them together.